test/test_prepare.exs

```diff
@@ -13,3 +13,20 @@ for file <- File.ls!(test_contracts_path) do
       ]
     )
 end
+
```

Comment:
We should not add this to test_prepare! We should move this to its own test setup in `multicall_test.exs` in the same `deploy_contracts` setup function.